src/core/Akka/Actor/Scheduler/HashedWheelTimerScheduler.cs

```diff
@@ -62,35 +82,37 @@ public HashedWheelTimerScheduler(Config scheduler, ILoggingAdapter log) : base(s
 
             // prevent overflow
             if (_tickDuration >= long.MaxValue / _wheel.Length)
+            {
+                // ReSharper disable NotResolvedInText
                 throw new ArgumentOutOfRangeException("akka.scheduler.tick-duration", _tickDuration,
                     $"akka.scheduler.tick-duration: {_tickDuration} (expected: 0 < tick-duration in ticks < {long.MaxValue / _wheel.Length}");
+                // ReSharper restore NotResolvedInText
+            }
 
-            _shutdownTimeout = SchedulerConfig.GetTimeSpan("akka.scheduler.shutdown-timeout", null);
+            _shutdownTimeout = SchedulerConfig.GetTimeSpan("akka.scheduler.shutdown-timeout", TimeSpan.Zero);
```

Comment:
So this will result in an instant shutdown?

Reply:
Yes, it would, because `TimeSpan.Zero` is an illegal value